www/src/components/navigation/navbar.astro

```diff
@@ -59,13 +60,17 @@ const navbarLinks: Array<{ href: string; label: string }> = [
         }
       </div>
     </div>
-    <div>
-      <SidebarToggle currentPage={currentPage} client:idle />
-    </div>
-    <div
-      class="hidden md:block shadow-sm dark:shadow-md shadow-slate-500 dark:shadow-slate-500/10 rounded-lg w-full max-w-xs"
-    >
-      <Search client:load />
+    <div class="flex flex-grow justify-end items-center">
+      <div class={clsx("m-8 text-center", { hidden: isLanding })}>
+        <ThemeToggleButton />
```

Comment:
Need to load the JS here:

```suggestion
        <ThemeToggleButton client:load />
```

Reply:
Fixed thanks for letting me know :sweat_smile: